### PR TITLE
[WIPTEST] Updates to Chargeback Assignments Class and Tests

### PR DESCRIPTION
**Pull request by jawatts**
{{pytest: cfme/tests/intelligence/chargeback/test_chargeback_long_term_rates.py  cfme/tests/intelligence/chargeback/test_resource_allocation.py cfme/tests/intelligence/reports/test_validate_chargeback_report.py cfme/tests/intelligence/test_chargeback_assignments.py cfme/tests/ssui/test_ssui_dashboard.py --use-provider complete --long-running}}

### CODE REVIEW

**Comment by mshriver** [comment]
This is part of resolving #7960

File: cfme/tests/ssui/test_ssui_dashboard.py [hunk 1]
```diff
@@ -54,23 +54,23 @@ def new_compute_rate(enable_candu):
...
 def assign_chargeback_rate(new_compute_rate):
     """Assign custom Compute rate to the Enterprise and then queue the Chargeback report."""
-    enterprise = cb.Assign(
-        assign_to="The Enterprise",
-        selections={
-            'Enterprise': {'Rate': new_compute_rate}
-        })
-    enterprise.computeassign()
-    enterprise.storageassign()
+    for klass in (cb.ComputeAssign, cb.StorageAssign):
```
Comment (mshriver) [inline cfme/tests/ssui/test_ssui_dashboard.py]: Consider creating a singular fixture for this somewhere in `cfme/fixtures/` to reduce the code duplication throughout these tests. Might need some tweaks to the `'Default'` test case with a simple fixture providing that string for test parametrization.

File: cfme/intelligence/chargeback/assignments.py [hunk 2]
```diff
@@ -113,32 +107,36 @@ def _fill(self, view):
         return view.fill(fill_details)
 
 
+class ComputeAssign(Assign):
+    TYPE = "Compute"
+
+
+class StorageAssign(Assign):
+    TYPE = "Assign"
```
[the remark this answers is not in the document]
Comment (jawatts) [reply cfme/intelligence/chargeback/assignments.py]: Should be, typo. Fixed

File: cfme/tests/containers/test_chargeback.py [hunk 3]
```diff
@@ -77,14 +77,14 @@ def gen_report_base(appliance, obj_type, provider, rate_desc, rate_interval):
...
-                'filter_show_costs': 'Project',
+                'filter_show_costs': 'Container Project',
                 'filter_provider': provider.name,
                 'filter_project': 'All Container Projects'
             }
         }
     elif obj_type == 'Image':
         data = {
-            'base_report_on': 'Chargeback Container Images',
+            'base_report_on': 'Chargeback for Images',
```
[the remark this answers is not in the document]
Comment (jawatts) [reply cfme/tests/containers/test_chargeback.py]: Not between 59z and 510z

**Pull request merged**